src/test/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImplTest.java

```diff
@@ -71,8 +71,8 @@ void invokesTaskWhenTaskIsNotLocked() {
 
         adminAutomatedTaskService.runAutomatedTask(1);
 
-        verify(automatedTaskRunner, times(1)).run(someAutomatedTask);
-        verify(auditApi, times(1)).record(AuditActivity.RUN_JOB_MANUALLY,"some-task-name");
+        verify(automatedTaskRunner, times(1)).run(someAutomatedTask, true);
```

Comment:
You have set isManual to true but is there a test where its set to false

Reply:
the is manual run false should be covered by the existing tests for automated tasks. As when it is not manually run we default this value to be false